src/pages/payment/Payment.tsx

```diff
@@ -0,0 +1,78 @@
+import { useParams } from 'react-router-dom';
+import { Elements } from '@stripe/react-stripe-js';
+import { ChangeEvent, FormEvent, useCallback, useEffect, useState } from 'react';
+import { Box, FormControl, FormControlLabel, FormLabel, Radio, RadioGroup } from '@mui/material';
+import { PAYMENT_OPTION } from './components/contants';
+import { PaymentForm } from './components/PaymentForm';
+import { stripePromise } from '../../config/stripe.config';
+import ButtonPrimary from '../../components/button/ButtonPrimary';
+import LoadingPrimary from '../../components/loader/LoadingPrimary';
+import usePostPaymentOptionMutation from '../../api/mutations/payment/usePostPaymentOptionMutation';
+import DataFetchError from '../../components/shared/DataFetchError';
+
+const Payment = () => {
+  const bookingId = useParams().id;
```

Comment:
use destruction assignment pls with default value, it will look better

`const { id: bookingId = '' } = useParams()`